vtipy/hra_na_doktora.txt

```diff
@@ -0,0 +1,2 @@
+﻿Žena povídá "manžel si včera chtěl hrát na doktora,
```

Comment:
Chybí dvojtečka. Přímou řeč uvozujeme velkým písmenem. Ještě by to chtělo zlepšit uvozovky. Také bych to všechno sloučil na jeden řádek.